src/freenet/clients/http/QueueToadlet.java

```diff
@@ -1652,7 +1664,17 @@ else if(secondFinalized && !firstFinalized)
 			contentNode.addChild("a", "id", "uncompletedDownload");
 			HTMLNode uncompletedContent = pageMaker.getInfobox("requests_in_progress", l10n("wipD", new String[]{ "size" }, new String[]{ String.valueOf(uncompletedDownload.size()) }), contentNode, "download-progressing", false);
 			if (advancedModeEnabled) {
-				uncompletedContent.addChild(createRequestTable(pageMaker, ctx, uncompletedDownload, new QueueColumn[] { QueueColumn.IDENTIFIER, QueueColumn.PRIORITY, QueueColumn.SIZE, QueueColumn.MIME_TYPE, QueueColumn.PROGRESS, QueueColumn.LAST_ACTIVITY, QueueColumn.PERSISTENCE, QueueColumn.FILENAME, QueueColumn.KEY, QueueColumn.COMPAT_MODE }, priorityClasses, advancedModeEnabled, false, "uncompleted-download", false, false));
+                uncompletedContent.addChild(
+                    createRequestTable(
+                        pageMaker, ctx, uncompletedDownload,
+                        new QueueColumn[] {
+                            QueueColumn.IDENTIFIER, QueueColumn.PRIORITY, QueueColumn.SIZE,
+                            QueueColumn.MIME_TYPE, QueueColumn.PROGRESS, QueueColumn.LAST_ACTIVITY,
+                            QueueColumn.LAST_FAILURE, QueueColumn.PERSISTENCE, QueueColumn.FILENAME,
```

Comment:
It's hard to tell what changed here when functional changes are combined with formatting ones.